src/components/network-modification-tree-pane.js

```diff
@@ -83,6 +88,43 @@ export const NetworkModificationTreePane = ({
 
     const width = useStoreState((state) => state.width);
     const prevTreeDisplay = usePreviousTreeDisplay(studyMapTreeDisplay, width);
+    const workingNode = useSelector((state) => state.workingTreeNode);
+    //we use this useRef to avoid to trigger on this depedencie workingTreeNode. avoid infinite loop
+    const workingNodeRef = useRef();
+    workingNodeRef.current = workingNode;
+
+    // change working node to ROOT node after deleting the current working node
+    const updateWorkingNodeToRootAfterDeleting = useCallback(() => {
+        const networkModificationTree = fetchNetworkModificationTree(studyUuid);
+
```

Comment:
Remove this fetch and use `NetworkModificationTreeModel` in the redux which is the TreeModel